flow/src/main/java/com/sequenceiq/flow/service/FlowService.java

```diff
@@ -139,6 +139,12 @@ public FlowCheckResponse getFlowChainState(String chainId) {
             flowCheckResponse.setHasActiveFlow(!completed("Flow chain", chainId, relatedChains, relatedFlowLogs));
             flowCheckResponse.setLatestFlowFinalizedAndFailed(isFlowInFailedState(relatedFlowLogs, failHandledEvents));
             setEndTimeOnFlowCheckResponse(flowCheckResponse, relatedFlowLogs);
+            // Get the latest flow information that will be used for error logging.
```

Comment:
Please omit the comments describing the code. Use appropriate method and variable names.

Reply:
Removed the comments. Thank you!